start_cerberus.py

```diff
@@ -161,17 +59,25 @@ def main(cfg):
         # Initialize clients
         if not os.path.isfile(kubeconfig_path):
             kubeconfig_path = None
-        initialize_clients(kubeconfig_path)
+        kubecli.initialize_clients(kubeconfig_path)
 
         # Start cerberus
         logging.info("Starting cerberus")
+        logging.info("Fetching cluster info")
+        cluster_version = runcommand.invoke("kubectl get clusterversion")
+        cluster_info = runcommand.invoke("kubectl cluster-info | awk 'NR==1'")
+        print("%s %s" % (cluster_version, cluster_info))
 
         # Run http server using a separate thread
         # if cerberus is asked to publish the status.
         # It is served by the http server.
         if cerberus_publish_status == "True":
-            logging.info("Publishing cerberus status at http://localhost:8086")
-            _thread.start_new_thread(start_server, ())
+            address = ("0.0.0.0", 8080)
```

Comment:
Should we allow the user to pass the port via the YAML? or maybe default to 8080 but allow it to be overwritten?